src/transformers/models/gpt2/configuration_gpt2.py

```diff
@@ -113,6 +114,11 @@ class GPT2Config(PretrainedConfig):
             Whether or not to use gradient checkpointing to save memory at the expense of slower backward pass.
         use_cache (:obj:`bool`, `optional`, defaults to :obj:`True`):
             Whether or not the model should return the last key/values attentions (not used by all models).
+        scale_attn_by_layer_idx (:obj:`bool`, `optional`, defaults to :obj:`False):
+            Whether to additionally scale attention weights by 1 / layer_idx.
```

Comment:
```suggestion
            Whether to additionally scale attention weights by ``1 / layer_idx + 1``.
```

add missing 1, add code format